server/data/accreditedProgrammesApi/referralClient.ts

```diff
@@ -23,6 +30,15 @@ export default class ReferralClient {
     })) as CreatedReferralResponse
   }
 
+  async dashboard(organisationId: string): Promise<Paginated<ReferralSummary>> {
```

Comment:
How about `findReferralSummaries(organisationId)` here? I'm having second thoughts on the `dashboard` label for the endpoint, as we're referring to it as `case-list` elsewhere, but I think naming it after the entity being returned might be most future-proof.

Reply:
Yeah I was unsure about the naming too, happy to go with `findReferralSummaries`